system/lib/wasmfs/syscalls.cpp

```diff
@@ -1123,6 +1119,16 @@ int __syscall_readlinkat(int dirfd,
   return bytes;
 }
 
+static double timespec_to_ms(timespec ts) {
+  if (ts.tv_nsec == UTIME_OMIT) {
+    return INFINITY;
```

Comment:
Where is INFINITY defined?

Reply:
[<math.h>: ](https://en.cppreference.com/w/c/numeric/math/INFINITY)

Reply:
Ah, thanks, what I was confused about is that `tv_nsec` is a `long`, and I missed that we return a different type here, `double`.